### PR TITLE
Bump version to 2.0.0

### PR DESCRIPTION
**Pull request by MasterOdin**
See changelog in diff for the changeset

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 2.0.0 (07/19/2021)
+
+* __breaking__ Drop support for Python 3.5 (thanks [@nebnola](https://github.com/nebnola))
```
Comment (marchersimon) [inline CHANGELOG.md:5]: I missed that PR and I don't really understand why we did that. Isn't Pyton 3.5 relatively new?

**Review by marchersimon — APPROVED** [review]
Looks good, btw

File: CHANGELOG.md (hunk 1 above)
Comment (MasterOdin) [reply CHANGELOG.md:5]: Python 3.5.0 was released 09/13/2015 and reached end-of-life in 09/20/2020. Assuming you're not running a very old LTS release (e.g. Ubuntu 16.04), you should be running minimally Python 3.6+.

The move to 3.6 does allow rewriting our string handling to use f-strings which is a nice boon in my mind, and again, no reason to hold ourselves back for the sake of EOL software.

Comment (marchersimon) [reply CHANGELOG.md:5]: Oh, thanks.

**Review by CleanMachine1 — APPROVED** [review]
Wording is fine.

**Comment by CleanMachine1** [comment]
Remember to make the tag.

**Comment by MasterOdin** [comment]
Please do give me more than 3 minutes to cut a release in the future before posting a reminder.

**Comment by CleanMachine1** [comment]
Sorry for that. I saw the branch name and thought without checking that you hadn't made one for 1.3.0.

**Pull request merged**